_aicspylibczi/pb_helpers.cpp

```diff
@@ -56,6 +55,8 @@ namespace pb_helpers {
       case libCZI::PixelType::Bgr96Float:
           arr = memoryToNpArray<float>(icBase, charSizes);
           break;
+      default:
+          throw pylibczi::PixelTypeException(icBase->pixelType(), "Unsuported pixel type in helper function.");
```

Comment:
does this get caught on the python side?

Reply:
`Unsupported`

Reply:
So it is currently not in a try/catch if it were it would get caught by that. What's the preferred behavior here?

Reply:
It really depends on a few things in general:   do you want to recover from as many errors as gracefully as you can, and forward them to "raise" on the python side with useful messages?  are there some errors that are too egregious to recover from?    I don't know if you have a master try-catch on the api surface where you go between python and c++.

Reply:
Same behavior as in Reader.cpp. I'll add a comment here too.